File-Formats/DocIO/Word-document/Compare Word documents.md

```diff
@@ -0,0 +1,229 @@
+
+' Open the original Word document.
+Using originalDocument As New WordDocument(originalFilePath, FormatType.Docx)
+    ' Open the revised Word document.
+    Using revisedDocument As New WordDocument(revisedFilePath, FormatType.Docx)
+        ' Compare original document with revised document.
+        originalDocument.Compare(revisedDocument, "Nancy Davolio", DateTime.Now.AddDays(-1))
+        ' Save the Word document.
+        originalDocument.Save(resultFilePath)
+    End Using
+End Using
+
+{% endhighlight %}
+{% endtabs %}
+
+![Compare Word documents in C#](WorkingwithWordDocument_images/Compare-Word-documents.png)
+
+## Comparison options
+
+You can customize the Word comparison using our ComparisonOptions in DocIO.
+
+### Ignore format changes
+
+In the .NET Word library (DocIO), document comparison includes formatting changes by default. However, you can configure DocIO to ignore formatting differences using DetectFormatChanges API to concentrate solely on content modifications.
+
+The following code example illustrates how to compare two Word documents by ignoring the format changes.
+
+
+{% tabs %}
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Loads the original document
+using (FileStream originalDocumentStreamPath = new FileStream("OriginalDocument.docx", FileMode.Open, FileAccess.Read))
+{
+    using (WordDocument originalDocument = new WordDocument(originalDocumentStreamPath, FormatType.Docx))
+    {
+        //Loads the revised document
+        using (FileStream revisedDocumentStreamPath = new FileStream("RevisedDocument.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+        {
+            using (WordDocument revisedDocument = new WordDocument(revisedDocumentStreamPath, FormatType.Automatic))
+            {
+                //Sets the Comparison option detect format changes, whether to detect format changes while comparing two Word documents
+                ComparisonOptions compareOptions = new ComparisonOptions();
+                compareOptions.DetectFormatChanges = false;
+                //Compares the original document with revised document
+                originalDocument.Compare(revisedDocument, "Syncfusion", DateTime.Now, compareOptions);
+                //Saves the Word document to MemoryStream
+                using (MemoryStream stream = new MemoryStream())
+                {
+                    originalDocument.Save(stream, FormatType.Docx);
+                }
+            }
+        }
+    }
+}
+
+{% endhighlight %}
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+//Loads the original document 
+using (WordDocument originalDocument = new WordDocument("OriginalDocument.docx"))
+{
+    //Loads the revised document
+    using (WordDocument revisedDocument = new WordDocument("RevisedDocument.docx"))
+    {
+        //Sets the Comparison option detect format changes, whether to detect format changes while comparing two Word documents
```

Comment:
//Set the Comparison option detect format changes, whether to detect format changes while comparing two Word documents.